AmazonChimeSDKDemo/AmazonChimeSDKDemo/TestSettingsViewController.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  TestSettingsViewController.swift
+//  AmazonChimeSDKDemo
+//
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+import UIKit
+
+class TestSettingsViewController: UIViewController, UITextFieldDelegate {
+    @IBOutlet var serverEndpointTextField: UITextField!
+    @IBOutlet var doneButton: UIButton!
+    
+    private let meetingPresenter = MeetingPresenter.shared()
```

Comment:
meetingPresenter should not be in this VC class

Reply:
Make sense.